src/panels/lovelace/cards/hui-entity-card.ts

```diff
@@ -0,0 +1,270 @@
+            : ""}
+        </div>
+        ${this._config.footer
+          ? this.renderHeaderFooter(this._config.footer, "footer")
+          : ""}
+      </ha-card>
+    `;
+  }
+
+  protected shouldUpdate(changedProps: PropertyValues): boolean {
+    return hasConfigOrEntityChanged(this, changedProps);
+  }
+
+  protected updated(changedProps: PropertyValues) {
+    super.updated(changedProps);
+    if (!this._config || !this.hass) {
+      return;
+    }
+
+    const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
+    const oldConfig = changedProps.get("_config") as
+      | EntityCardConfig
+      | undefined;
+
+    if (
+      !oldHass ||
+      !oldConfig ||
+      oldHass.themes !== this.hass.themes ||
+      oldConfig.theme !== this._config.theme
+    ) {
+      applyThemesOnElement(this, this.hass.themes, this._config!.theme);
+    }
+  }
+
+  private renderHeaderFooter(
+    conf: LovelaceHeaderFooterConfig,
+    className: string
+  ): TemplateResult {
+    const element = createHeaderFooterElement(conf);
+    if (this.hass) {
+      element.hass = this.hass;
+    }
+    return html`
+      <div class=${"header-footer " + className}>${element}</div>
+    `;
+  }
+
+  private _handleClick(): void {
+    fireEvent(this, "hass-more-info", { entityId: this._config!.entity });
+  }
+
+  static get styles(): CSSResult {
+    return css`
+      :host {
+        display: flex;
+        flex-direction: column;
+      }
+
+      ha-card {
+        display: flex;
+        flex-direction: column;
+        flex: 1;
+        position: relative;
+        cursor: pointer;
+        overflow: hidden;
+      }
+
+      ha-card:focus {
+        outline: none;
+        background: var(--divider-color);
+      }
+
+      .flex {
+        display: flex;
+      }
+
+      .header {
+        margin: 8px 16px 0;
+        justify-content: space-between;
+      }
+
+      .name {
+        align-items: center;
+        display: flex;
+        min-width: 0;
+        opacity: 0.8;
+        position: relative;
+      }
+
+      .name > span {
+        display: block;
+        display: -webkit-box;
+        font-size: 1.2rem;
+        font-weight: 500;
+        max-height: 1.4rem;
+        top: 2px;
+        opacity: 0.8;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        -webkit-line-clamp: 1;
+        -webkit-box-orient: vertical;
+        word-wrap: break-word;
+        word-break: break-all;
+      }
+
+      .icon {
+        color: var(--paper-item-icon-color, #44739e);
+        line-height: 40px;
+      }
+
+      .info {
+        flex-wrap: wrap;
+        margin: 0 16px 16px;
+      }
+
+      #value {
+        display: inline-block;
+        font-size: 2rem;
+        font-weight: 400;
+        line-height: 1em;
+        margin-right: 4px;
+      }
+
+      #measurement {
+        align-self: flex-end;
+        display: inline-block;
+        font-size: 1.3rem;
+        line-height: 1.2em;
+        margin-top: 0.1em;
```

Comment:
Why do this in em? Are you trying to make this card grow proportionally when the user increases font size?

Reply:
I didn't make this decision 🙂